tests/components/local_file/test_camera.py

```diff
@@ -14,7 +14,7 @@ async def test_loading_file(hass, hass_client):
 
     with mock.patch("os.path.isfile", mock.Mock(return_value=True)), mock.patch(
         "os.access", mock.Mock(return_value=True)
-    ):
+    ), mock.patch("mimetypes.guess_type", mock.Mock(return_value=(None, None))):
```

Comment:
```suggestion
    ), mock.patch("homeassistant.components.local_file.camera.mimetypes.guess_type", mock.Mock(return_value=(None, None))):
```

Can we patch this where its imported so we don't end up doing it globally?

Reply:
I have adjusted - but I'm still confused as to why it only fails on partial CI

Reply:
Maybe something else is patching it globally and not resetting the patch ?

Reply:
Possibly but I couldn't find it!